Changelog.md

```diff
@@ -20,6 +20,7 @@ to get an offset of the error position. For `SyntaxError`s the range
 
 ### New Features
 
+- [#721]: Report bytes from `Writer::write_event`
```

Comment:
Could you make this 

> Report the number of bytes written from ...

Just so that it's more obvious what it's referring to